### ISSUE TITLE
consider adding decTalk TTS to NVDA 

### ISSUE THREAD

**Issue by joshknnd1982** [nvaccess/nvda#12467]
Would you please consider adding the DecTalk tts to NVDA as an alternate tts engine, perhaps making it sapi5, ? or putting it on the NVDA extra voices page? I learned that Fonix or speech-fx-inc www.speechfxinc.com is dead, abandoned and its domain up for sale, and decTalk source code was released by one of the developers before he died, found here... 

http://grossgang.com/tts/dectalk%20software%20and%20manual/Ad%202.zip

There is also a decTalk bundle for NVDA addon works with python3 NVDA here, perhaps you could verify its legality and if its ok and legal, add it to NVDA's extra voices, make it sapi5 or include it in future NVDA versions? 

http://grossgang.com/tts/synthesizers%20for%20nvda/dectalk/DECtalk%20py3%20bundle.nvda-addon

**Comment by zstanecic** [comment]
Hi Josh,

This synthesizer NVDA addon should be ported to NVDA 2021.1. 

Best,

Zvonimir

 

From: joshknnd1982 ***@***.***> 
Sent: Wednesday, May 26, 2021 10:54 PM
To: nvaccess/nvda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nvaccess/nvda] consider adding decTalk TTS to NVDA (#12467)

 

Would you please consider adding the DecTalk tts to NVDA as an alternate tts engine, perhaps making it sapi5, ? or putting it on the NVDA extra voices page? I learned that Fonix or speech-fx-inc www.speechfxinc.com <http://www.speechfxinc.com>  is dead, abandoned and its domain up for sale, and decTalk source code was released by one of the developers before he died, found here...

http://grossgang.com/tts/dectalk%20software%20and%20manual/Ad%202.zip

There is also a decTalk bundle for NVDA addon works with python3 NVDA here, perhaps you could verify its legality and if its ok and legal, add it to NVDA's extra voices, make it sapi5 or include it in future NVDA versions?

http://grossgang.com/tts/synthesizers%20for%20nvda/dectalk/DECtalk%20py3%20bundle.nvda-addon

**Comment by cary-rowen** [comment]
This synthesizer does not have multi-language support and I am afraid it will not be integrated into NVDA.

**Comment by feerrenrut** [comment]
I'm closing this issue, it is not specific to the NVDA project. Please reopen an issue once it has been verified that it is feasible (legally and technically) to integrate the TTS. Even then, if the TTS does not support multiple languages, it is better to remain an addon.
